doc/uploads/plantuml/ara_sequence_diagram.puml

```diff
@@ -0,0 +1,18 @@
+@startuml
+group upload zip (ExecutionRessource)
+  ARA <- CI : sending ZIP
```

Comment:
You should reverse this (`CI -> ARA`) : To change the orientation of the image and make it easy to understand as a left-to-right layers

Reply:
done